src/types/observation.js

```diff
@@ -23,4 +23,5 @@ export interface Observation {
   icon: any;
   categoryId: string;
   fields: Field[];
+  createdFrom: string;
```

Comment:
It may be worth it to put this just in the AppState instead of on the observation model and then we can set it as undefined when we set selectedObservation as undefined. That way we keep the observation model pure and reflective of the API.

Reply:
Ahh true